src/index.html

```diff
@@ -14,8 +14,55 @@
     >
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto&display=swap">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="mainCard" data-qa="card">
+      <img class="mainCard__photo"
+        src="images/imac.jpeg" alt="mac">
+
+      <div class="mainCard__description">
```

Comment:
remove this div

Reply:
you mean this   <div class="mainCard__description"> ? I can't remove it

Reply:
you mean to remove <div class="mainCard__description"> ? i cant remove it